challenger-deep-theme.el

```diff
@@ -383,12 +383,12 @@
 				    (,ct (:background, ct-blue-d :foreground, ct-black))))
 
      ;; diff-hl
-     `(diff-hl-change              ((,c (:foreground ,vc-modified))
-                                    (,ct (:foreground ,ct-vc-modified))))
-     `(diff-hl-delete              ((,c (:foreground ,vc-deleted))
-                                    (,ct (:foreground ,ct-vc-deleted))))
-     `(diff-hl-insert              ((,c (:foreground ,vc-added))
-                                    (,ct (:foreground ,ct-vc-added))))
+     `(diff-hl-change              ((,c (:foreground ,vc-modified :background ,vc-modified))
+                                    (,ct (:foreground ,ct-vc-modified :background ,ct-vc-modified))))
+     `(diff-hl-delete              ((,c (:foreground ,vc-deleted :background ,vc-deleted))
+                                    (,ct (:foreground ,ct-vc-deleted :background ,ct-vc-deleted))))
+     `(diff-hl-insert              ((,c (:foreground ,vc-added :background ,vc-added))
+                                    (,ct (:foreground ,ct-vc-added :background ,vc-added))))
```

Comment:
> Btw, does the t inside ct- means terminal? I have not tested it on a
16-color terminal, but it works fine on my 256-color terminal and true-color
terminal.

Yes, ct means 256 color terminal, so we should use the ct colors here
```suggestion
                                    (,ct (:foreground ,ct-vc-added :background ,ct-vc-added))))
```

Reply:
I see, thanks for explanation. Sorry I forgot to add the prefix, I'll add them now.